ozaria/site/views/play/level/PlayLevelView.js

```diff
@@ -335,6 +336,17 @@ class PlayLevelView extends RootView {
 
   grabLevelLoaderData () {
     this.session = this.levelLoader.session
+    console.log('CAPSTONE: grabLevelLoaderData with : ')
+    console.log(this.capstoneStage)
+    console.log(me.isSessionless())
+
+    // if (this.capstoneStage && me.isSessionless() && (me.isAdmin() || me.isTeacher())) {
```

Comment:
Line required?

Reply:
Updating with a comment here to explain that we are commenting this out for Ozaria launch 👍 

We want these restrictions in the future, but for now we only want this to be possible for admins.